lib/shared/homes_guests_love_card.dart

```diff
@@ -0,0 +1,142 @@
+import 'package:flutter/material.dart';
+import '../../../theme.dart';
+
+class HomeGuestsLoveCard extends StatelessWidget {
+  final bool isHorizontalList;
+  final String id;
+  final String imageUrl;
+  final String title;
+  // final String shortDescription;
+  // final String longDescription;
+  final String location;
+  // final String postalCode;
+  final int price;
+  final int categorization;
+  // final int capacity;
+  // final String accommodationType;
+  // final bool freeCancelation;
+
+  HomeGuestsLoveCard({
+    required this.isHorizontalList,
+    required this.id,
+    required this.imageUrl,
+    required this.title,
+    // required this.shortDescription,
+    // required this.longDescription,
+    required this.location,
+    // required this.postalCode,
+    required this.price,
+    required this.categorization,
+    // required this.capacity,
+    // required this.accommodationType,
+    // required this.freeCancelation,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final List<Widget> starsRating = [];
+    for (int i = 0; i < categorization; i++) {
+      starsRating.add(Icon(
```

Comment:
You can separate this to new widget. Widget that will return list of stars depending on categorization.